symphony-bdk-core/build.gradle

```diff
@@ -48,6 +48,7 @@ dependencies {
     implementation 'org.bouncycastle:bcpkix-jdk15on'
     api 'com.fasterxml.jackson.core:jackson-databind'
     api 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml'
+    api 'com.fasterxml.jackson.dataformat:jackson-dataformat-properties'
```

Comment:
I'm not sure we should be exposing this api (but as implementation instead) because it is not exposed in the public API of this module, but it is already the case for the yaml module (and if we change it now it could probably break existing bots that do not declare this dependency and use it transitively...)